aiidalab_widgets_base/bug_report.py

```diff
@@ -111,11 +111,11 @@ def _convert_ansi_codes_to_html(msg):
 def _format_truncated_traceback(traceback, max_num_chars=2000):
     """Truncate the traceback to the given character length."""
     n = 0
-    for i, line in enumerate(reversed(traceback)):
+    for _i, line in enumerate(reversed(traceback)):
```

Comment:
I don't understand this change, it looks very confusing to me. Which rule is violated here?

Reply:
The format checker said something like: "`i` was not used inside the loop, so add an underscore in front."

Reply:
Fair, but still looks very confusing to me, never seen a variable (not method) prefixed with underscore. Maybe call it `lineno` or `line_number`?

Reply:
> Fair, but still looks very confusing to me, never seen a variable (not method) prefixed with underscore. Maybe call it `lineno` or `line_number`?

You meant `_lineno`/`line_number`? I don’t think the name without an underscore will satisfy the checks.

Here are a few words about the use of underscore for the variable names: https://medium.com/python-features/naming-conventions-with-underscores-in-python-791251ac7097

Reply:
Ah, I misunderstood then. So be it, but I'd still use a more descriptive variable name to make things clearer. Thanks!

In the article they explain the use the use of a leading underscore to indicate private vars / methods, which I was familiar with, but I don't think that is the case here so it still looks odd to me tbh.

Reply:
You are right. In the end, this is only to satisfy the check. Probably, this part of the code could be written better.